src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/UnallocatedDataBaseOperationService.kt

```diff
@@ -46,10 +47,15 @@ class UnallocatedDataBaseOperationService(
       .filter { !activeEvents.containsKey(it.convictionNumber) }
       .forEach { deleteEvent ->
         logger.debug("Deleting event for CRN: ${deleteEvent.crn}, conviction number: ${deleteEvent.convictionNumber}, teamCode: ${deleteEvent.teamCode}")
-        repository.delete(deleteEvent)
-        logger.debug("Event $deleteEvent deleted")
-        val team = workforceAllocationsToDeliusApiClient.getAllocatedTeam(deleteEvent.crn, deleteEvent.convictionNumber)
-        telemetryService.trackUnallocatedCaseAllocated(deleteEvent, team?.teamCode)
+        try {
+          repository.delete(deleteEvent)
+          logger.debug("Event {} deleted", deleteEvent)
```

Comment:
We should perhaps stick to the established convention in codebase of  `logger.debug("Event $deleteEvent deleted")`

Reply:
sorted

Reply:
sorted